### PR TITLE
Add ability to move map to user-entered coordinates using latitude and longitude text fields and button

### PR DESCRIPTION
**Pull request by Asheshp23**
Added latitude and longitude text fields and a button that allows the user to move the map to the entered coordinates.

https://github.com/twostraws/ControlRoom/assets/22404192/5ad8b96c-7153-4e35-a8e5-e1c0191bc735

### CODE REVIEW

File: ControlRoom/Simulator UI/ControlScreens/LocationView.swift [hunk 1]
```diff
@@ -8,12 +8,15 @@
...
 /// Map view to change simulated user's position
 struct LocationView: View {
     @ObservedObject var controller: SimulatorsController
     let simulator: Simulator
 
+    @State private var latitudeText = "37.323056"
+    @State private var longitudeText = "-122.031944"
     /// The location that is being simulated
     @State private var currentLocation = MKCoordinateRegion(
         center: CLLocationCoordinate2D(latitude: 37.323056, longitude: -122.031944),
```
Comment (ipranay) [inline ControlRoom/Simulator UI/ControlScreens/LocationView.swift:22]: Might want to use constants here and in lines 18, 19 to easily change those in the future.

e.g.

```
DEFAULT_LAT = 37.323056
DEFAULT_LNG = -122.031944
```

18, 19 become:
```
    @State private var latitudeText = "\(DEFAULT_LAT)"
    @State private var longitudeText = "\(DEFAULT_LNG)"
```

```suggestion
        center: CLLocationCoordinate2D(latitude: DEFAULT_LAT, longitude: DEFAULT_LNG),
```

Comment (Asheshp23) [reply ControlRoom/Simulator UI/ControlScreens/LocationView.swift:22]: We cannot use instance member 'DEFAULT_LAT' within property initializer; property initializers run before 'self' is available.

**Review by ipranay — APPROVED** [review]

**Comment by Asheshp23** [comment]
@twostraws We've added latitude and longitude text fields and a button that allows the user to move the map to the entered coordinates.

Could you please review it and merge it?

**Comment by twostraws** [comment]
Great job – thank you very much!

**Pull request merged**